kennel.gemspec

```diff
@@ -15,5 +15,6 @@ Gem::Specification.new name, Kennel::VERSION do |s|
   s.add_runtime_dependency "faraday", "~> 1.8"
   s.add_runtime_dependency "hashdiff", "~> 1.0"
   s.add_runtime_dependency "net-http-persistent", "~> 4.0"
+  s.add_runtime_dependency "rosarium", "~> 0.1"
```

Comment:
Does this need to be added to `Gemfile` too? I've never understood why there's both a `Gemfile` and a gemspec, both specifying the dependencies.

Reply:
gemspec is for what the gem is dependent on
Gemfile is for what the development environment is dependent on

Reply:
TIL :-)